ci/jsonnet-mixin.tmpl

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
This file is the template that the `build.sh` uses to build the Jsonnet file. It just substitutes out `[/*REMOTE_WRITE*/]` with JSON-ified prometheus.prometheusSpec.remoteWrite.

I didn't know where to put this file since there doesn't appear to be anything like it, so let me know if it needs to be moved somewhere.